scraper/src/password.py

```diff
@@ -0,0 +1,41 @@
+from base64 import b64encode, b64decode
+from cryptography.fernet import Fernet
+from hashlib import sha256
+from binascii import Error as BinAsciiError
+
+from logger import log
+
+class SymmetricEncryption:
+	def __init__(self):
+		# Change the following line as desired
+		password = "my_super_secret_password___BtpdG8VBwSzGPkCiKKTDfvlXamRy7K6rfs/NGqxjOOY="
```

Comment:
Even though this is only the key for the encryption, wouldn't it be better to have it on a separate file instead of in the source code itself?

Reply:
I see your point. It was just a hack, but if we're going with this, it would indeed be better to put this password in a separate file. I'll put in the next PR as well.